webpack.config.js

```diff
@@ -7,22 +7,33 @@ module.exports = function (webpackConfig, env) {
   webpackConfig.babel.plugins.push('transform-runtime');
   webpackConfig.babel.plugins.push(['import', {
     libraryName: 'antd',
-    style: 'css'  // if true, use less
+    style: true,
   }]);
 
   // Enable hmr for development.
   if (env === 'development') {
     webpackConfig.babel.plugins.push('dva-hmr');
   }
-
+  
+  // CSS Modules Support.
   // Parse all less files as css module.
-  webpackConfig.module.loaders.forEach(function (loader, index) {
+  webpackConfig.module.loaders.forEach(function(loader, index) {
     if (typeof loader.test === 'function' && loader.test.toString().indexOf('\\.less$') > -1) {
-      loader.test = /\.dont\.exist\.file/;
+      loader.include = /node_modules/;
+      loader.test = /\.less$/;
     }
     if (loader.test.toString() === '/\\.module\\.less$/') {
+      loader.exclude = /node_modules/;
       loader.test = /\.less$/;
     }
+    if (typeof loader.test === 'function' && loader.test.toString().indexOf('\\.css$') > -1) {
+      loader.include = /node_modules/;
+      loader.test = /\.css$/;
+    }
+    if (loader.test.toString() === '/\\.module\\.css$/') {
+      loader.exclude = /node_modules/;
+      loader.test = /\.css$/;
+    }
```

Comment:
copy it from https://github.com/dvajs/dva-cli/blob/085a58290b9580a15614184bcfd5fed1a945ed11/boilerplates/demo/webpack.config.js#L15-L34